src/main/java/frc/robot/Limelight.java

```diff
@@ -22,4 +62,202 @@ public void togglePipeline() {
         }
 
     }
+
+    /**
+     * Gets the horizontal offset angle from networkTable
+     * @return The horizontal offset angle from the limelight to the target
+     */
+    public Double getHorizontalOffset() {
+        
+        if(hasTargets()){
+
+        return table.getEntry("tx").getNumber(0).doubleValue();
+        
+        }
+        return Double.NaN;
+    }
+
+   
+
+    /**
+     * Gets the vertical offset angle from the limelight to the target
+     * @return The vertical angle from the limelight to the target
+     */
+    public Double getVerticalOffset() {
+
+        if(hasTargets()){
+
+            return table.getEntry("tx").getNumber(0).doubleValue();
+
+        }
+            return Double.NaN;
+
+    }
+
+   
+
+     /**
+      * Gets the skew of the robot from network tables
+      * @return The skew of the robot from -90 to 90 degrees
+      */
+    public Double getSkew() {
+
+        if(hasTargets()){
+
+            return table.getEntry("ts0").getNumber(0).doubleValue();
+
+        }
+            return Double.NaN;
+        
+        
+    }
+
+   
+
+     /**
+      * Returns the total area that the current target takes up on the limelight's screen
+      * Outputs a value associated with the percent of the screen being taken up
+      * @return The area of the limelight screen being taken up
+      */
+    public Double getTargetArea() {
+
+        if(hasTargets()){
+        return table.getEntry("ta").getNumber(0).doubleValue();
+        }
+
+        return Double.NaN;
+    }
+
+
+     /**
+      * Uses the horizontal offset that determines if the robot is looking left
+      * @return A boolean that says if the robot is looking left
+      */
+    public boolean isLookingLeft() {
+
+        return getHorizontalOffset() < 0;
+
+    }
+
+    /**
+     * Uses the pitch and yaw values from networkTables to construct and returns a rotation2D
+     * @return A rotation2D made from the robot pitch and yaw values
+     */
```

Comment:
elaborate on the fact that this is a rotation2d from the robot to the detected target